scripts/config/org/icepear/echarts/origin/chart/treemap/treemap-series-option.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "TreemapSeriesOption",
+    "type": "interface",
+    "extends": [
+        "SeriesOption",
+        "TreemapStateOption",
+        "BoxLayoutOptionMixin",
+        "RoamOptionMixin",
+        "TreemapSeriesVisualOption"
+    ],
+    "fields": [
+        { "name": "type", "types": ["String"], "default": "treemap"},
+        { "name": "size", "types": ["Number[]", "String[]"]},
+        { "name": "sort", "types": ["Boolean"], "default": true},
+        { "name": "clipWindow", "types": ["String"]},
+        { "name": "squareRatio", "types": ["Number"]},
+        { "name": "leafDepth", "types": ["Number"]},
+        { "name": "drillDownIcon", "types": ["String"]},
+        { "name": "zoomToNodeRatio", "types": ["Number"]},
+        { "name": "nodeClick", "types": ["String"]},
+        { "name": "breadcrumb", "types": ["TreemapSeriesBreadCrumbOption"]},
+        { "name": "levels", "types": ["TreemapSeriesLevelOption"]},
```

Comment:
I think it should be an array. https://github.com/apache/echarts/blob/566ba8e83dd283ba359fe9aceee0d4a12dbcd6c7/src/chart/treemap/TreemapSeries.ts#L210